umetnine/umetnine/settings.py

```diff
@@ -135,12 +138,12 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/3.0/howto/static-files/
 
-STATIC_URL = '/static/'
+STATIC_URL = FORCE_SCRIPT_NAME + 'static/'
 
 CRISPY_TEMPLATE_PACK = 'bootstrap4'
 
-LOGIN_REDIRECT_URL = '/user/profile'
-LOGOUT_REDIRECT_URL = '/user/logout'
+LOGIN_REDIRECT_URL = FORCE_SCRIPT_NAME + 'user/profile'
+LOGOUT_REDIRECT_URL = FORCE_SCRIPT_NAME + 'user/logout'
```

Comment:
Tukaj nisem prepričan, ali je to res potrebno. Če preusmerjanje po prijavi in odjavi ne bo delovalo pravilno, potem vrnita ti dve vrstici na prejšnje stanje.